docs/source/cluster_management/sweep.rst

```diff
@@ -30,23 +28,54 @@ Reasons to Sweep
 2. Improving Performance
     - Making many edits to the same row will leave behind many tombstoned entries, so it's advantageous to sweep these tables to increase performance.
 
+3. Improving Stability
+    - We have seen situations in the field where reading rows with many historical values caused the underlying Cassandra KVS to run out of memory.
+      This situation could have been mitigated by sweeping the table in question.
+
 Ways to Sweep
 -------------
 
-Under normal use, the sweep job is intended to run at a constant interval as a background process that does not consume significant system resources.
-Older AtlasDB instances with high data scale and months/years of user activity will likely have a large number of accumulated cells that are eligible for sweeping (i.e. a lot of unused data in AtlasDB that has never been swept).
-In this case, consider manually sweeping specific tables to reduce the number of unused cells that have accumulated over time.
+- :ref:`Background sweep process<background_sweep>` scheduled periodically:
```

Comment:
may be outside the scope of this PR, but we don't actually say how to enable background sweep, and it would be great to add that as well.